vxfreeswitch/voice.py

```diff
@@ -336,10 +341,17 @@ def register_client(self, client):
 
     def deregister_client(self, client, duration=None):
         client_addr = client.get_address()
+
+        # If originated call has not yet been answered
+        if self._unanswered_channels.get(client_addr):
+            d = self._unanswered_channels[client_addr]
+            d.errback(FreeSwitchClientError('Call is unanswered'))
```

Comment:
The problem is, in `handle_outbound_message`, we don't have the client available, since the client hasn't connected yet. When the client connects, we have no idea whether the connection was from dialling outbound or from an inbound connection, so we can't tell if we need to wait for the client to answer or not.